cds/modules/deposit/api.py

```diff
@@ -221,6 +221,7 @@ def commit(self, **kwargs):
         """Set partial validator as default."""
         if 'validator' not in kwargs:
             kwargs['validator'] = PartialDraft4Validator
+        # TODO add 'modified_by' inside _cds
```

Comment:
Probably we should add `modified_by` for audit as we are doing on [migration](https://github.com/CERNDocumentServer/cds/blob/8d4d43068f37901d7d85bbb5c22df4e51a403b8b/cds/modules/migrator/records.py#L227).
But I preferred wait to discuss if/how IRL before procede.